scrapers/Xvideos.yml

```diff
@@ -14,11 +33,20 @@ xPathScrapers:
       Performers:
         Name: //li[@class="model"]/a/span[1]/text()
       Studio:
-        Name: //a[@class="btn btn-default label main uploader-tag hover-name"]/span[@class="name"]/text()
+        Name: //li[@class="main-uploader"]/a[starts-with(@href,'/channels')]/span[@class="name"]/text()
```

Comment:
this breaks scenes where the studio is an amateur eg `https://www.xvideos.com/video75110099/real_amateur_milf_gets_passionate_pounding_in_her_lingerie`
do we get any false matches if we relax the selector a bit  ? eg 
```yaml
        Name: //li[@class="main-uploader"]/a/span[@class="name"]/text()
```

Reply:
>do we get any false matches if we relax the selector a bit ? (...)

Yes, if original poster is any random person having just a *profile*, he/she gets considered as a real studio.
This is why I added precision with *channels* only in the xpath.

Beside the amateur channel you provided (@bnkai), they are other channels types :
- channels
https://www.xvideos.com/video74652617/english_tutor_camilla_seduces_her_pupil_john_luna_with_her_big_natural_tits
- amateur-channels
https://www.xvideos.com/video75110099/real_amateur_milf_gets_passionate_pounding_in_her_lingerie
- model-channels 
https://www.xvideos.com/video75149889/29183767/0/all-natural_jessica_starling_gets_passionate_sex_from_johnny_sins
- pornstar-channels
https://www.xvideos.com/video74936872/mia_khalifa_-_trying_anal_it_s_never_going_to_happen_with_english_subtitles_
- ??-channels
...

Xvideos FAQ indicates there can be amateurs, porn stars,webcam models and erotic models account statuses.
*Model : model accounts can be amateurs, porn stars, webcam models, and erotic models. They indicate that you are the main content for your own account* (https://info.xvideos.com/account-status)
I wasn't able to find any other types of channels given this information.

So we are left with the following options so far :
1. Accept anything (original option) with the drawback of entering random person as real studio
2. Accept everything except random person (ie *profiles*) with the drawback of leaving the door opened for accepting unexpected "thing" as real studio
3. Be flexible by accepting anything matching `(?:.*-)?channels` with the same drawback as in (2.)
4. Be restrictive by accepting only the above listed channels types leaving the opportunity to add later newly discovered "thing" 
5. Any other option...

Reply:
Amateur channels are now supported. Addionnaly, model and pornstart channels are also supported.